video/out/w32_common.c

```diff
@@ -784,8 +785,10 @@ static void fit_window_on_screen(struct vo_w32_state *w32, RECT *rc)
 
     MP_VERBOSE(w32, "requested window size: %d:%d\n", o_w, o_h);
 
-    const int screen_w = w32->screenrc.x1 - w32->screenrc.x0;
-    const int screen_h = w32->screenrc.y1 - w32->screenrc.y0;
+    // Use virtual screen size, since the mpv window can be resized
+    // to cover mupltiple monitors
+    const int screen_w = GetSystemMetrics(SM_CXVIRTUALSCREEN);
+    const int screen_h = GetSystemMetrics(SM_CYVIRTUALSCREEN);
```

Comment:
I don't think mpv should use the virtual screen size here, since this will affect the window size with newly opened files as well as when mpv leaves fullscreen. Maybe mpv should just skip the fit-to-screen logic when leaving fullscreen, but use it in all other cases.

Reply:
The old behavior also affected newly opened files as well as leaving the fullscreen state, I'm not sure how using virtual size making things worse than now, can you elaborate please? I agree that skipping that function when leaving fullscreen might be a good idea. Should I leave the virtual screen size here anyway?